keyboards/flygone60/rev3/keymaps/via/keymap.c

```diff
@@ -0,0 +1,86 @@
+/* Copyright 2021 ShandonCodes
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Defines names for use in layer keycodes and the keymap
+enum layer_names {
+    _0,
+    _1,
+    _2,
+    _3
+};
+
+// Defines the keycodes used by our macros in process_record_user
+enum custom_keycodes {
+    SHILL = USER00,
+    NUT,
+    FRACNO
```

Comment:
The usefulness of these predefined macros in the `via` keymap is also questionable, but at least you used `USER00` correctly (VIA depends on the specific numeric values for keycodes and cannot use custom keycodes starting from `SAFE_RANGE`).

Reply:
I made the USER00 change due to the suggestion of @drashna. Also via docs are not exactly the most verbose, so I think there are some questionable conventions.